app/views/events/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Upcoming events</h1>
+<ul id='events'>
+  <% event = Event.upcoming_events.each do |event| %>
+    <li><%= link_to event.even_name, event %>
+      <%= event.date %>
+      <% if current_user %>
+      <%= link_to 'Attend', event_attend_path(event.id), method: :post %>
+      <% end %>
+    </li>
+  <% end %>
+</ul>
+
+<h1>Past Events</h1>
+<ul>
+  <%event = Event.past_events.each do |event| %>
```

Comment:
**REQUIRED**
There should be no logic in the views. 
We can have `upcoming_events` as a class instance variable in EventsController#index action and access it directly from the views.
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright